### ISSUE TITLE
object id is not generated in sequence

### ISSUE THREAD

**Issue by ancymicheal** [cfotache/pytorch_objectdetecttrack#12]
initiallly for one or two objects the id is generated in sequence later the is random as 8,21..... how to solve this

**Comment by smalldroid** [comment]
Yes, i have same problem. Object ids are generated not in sequence. And in object_detect.py, what is the purpose of n_cls_preds? Thank you!
unique_labels = detections[:, -1].cpu().unique()
        n_cls_preds = len(unique_labels)
